<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Synchro Time</string>
  </property>
  <property name="windowIcon">
   <iconset resource="synchrotime.qrc">
    <normaloff>:/images/icon.png</normaloff>:/images/icon.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="spacing">
     <number>5</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>3</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="Console" name="console" native="true"/>
    </item>
    <item>
     <widget class="QCustomPlot" name="customPlotBars" native="true">
      <property name="minimumSize">
       <size>
        <width>75</width>
        <height>200</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>80</width>
        <height>16777215</height>
       </size>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>640</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuConnect">
    <property name="title">
     <string>Connect</string>
    </property>
    <addaction name="actionConnect"/>
    <addaction name="actionDisconnect"/>
    <addaction name="separator"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuRequest">
    <property name="title">
     <string>Request</string>
    </property>
    <addaction name="actionInformation"/>
    <addaction name="actionAdjustment"/>
    <addaction name="actionCalibration"/>
    <addaction name="actionReset"/>
    <addaction name="separator"/>
    <addaction name="actionSetRegister"/>
   </widget>
   <widget class="QMenu" name="menuTools">
    <property name="title">
     <string>Tools</string>
    </property>
    <widget class="QMenu" name="menuSelect_Language">
     <property name="title">
      <string>Select Language</string>
     </property>
     <property name="icon">
      <iconset resource="synchrotime.qrc">
       <normaloff>:/images/edit.png</normaloff>:/images/edit.png</iconset>
     </property>
     <addaction name="actionEnglish"/>
     <addaction name="actionGerman"/>
     <addaction name="actionRussian"/>
    </widget>
    <addaction name="actionPort_Setting"/>
    <addaction name="actionClean_Up"/>
    <addaction name="actionSelect_Font"/>
    <addaction name="separator"/>
    <addaction name="menuSelect_Language"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionContents"/>
    <addaction name="actionAbout_Qt"/>
    <addaction name="separator"/>
    <addaction name="actionAbout_App"/>
   </widget>
   <addaction name="menuConnect"/>
   <addaction name="menuRequest"/>
   <addaction name="menuTools"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionConnect"/>
   <addaction name="actionDisconnect"/>
   <addaction name="separator"/>
   <addaction name="actionInformation"/>
   <addaction name="separator"/>
   <addaction name="actionPort_Setting"/>
   <addaction name="separator"/>
   <addaction name="actionClean_Up"/>
   <addaction name="separator"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionConnect">
   <property name="icon">
    <iconset resource="synchrotime.qrc">
     <normaloff>:/images/connect.png</normaloff>:/images/connect.png</iconset>
   </property>
   <property name="text">
    <string>C&amp;onnect</string>
   </property>
   <property name="toolTip">
    <string>Connect to serial port</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionDisconnect">
   <property name="icon">
    <iconset resource="synchrotime.qrc">
     <normaloff>:/images/disconnect.png</normaloff>:/images/disconnect.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Disconnect</string>
   </property>
   <property name="toolTip">
    <string>Disconnect from serial port</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+D</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="icon">
    <iconset resource="synchrotime.qrc">
     <normaloff>:/images/exit.png</normaloff>:/images/exit.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Quit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionInformation">
   <property name="icon">
    <iconset resource="synchrotime.qrc">
     <normaloff>:/images/info.png</normaloff>:/images/info.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Information</string>
   </property>
   <property name="toolTip">
    <string>Read information from RTC</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+I</string>
   </property>
  </action>
  <action name="actionAdjustment">
   <property name="icon">
    <iconset resource="synchrotime.qrc">
     <normaloff>:/images/clock.png</normaloff>:/images/clock.png</iconset>
   </property>
   <property name="text">
    <string>Adjustment</string>
   </property>
   <property name="toolTip">
    <string>Adjust the time</string>
   </property>
  </action>
  <action name="actionCalibration">
   <property name="icon">
    <iconset resource="synchrotime.qrc">
     <normaloff>:/images/stopwatch.png</normaloff>:/images/stopwatch.png</iconset>
   </property>
   <property name="text">
    <string>Calibration</string>
   </property>
   <property name="toolTip">
    <string>Calibrate the RTC</string>
   </property>
  </action>
  <action name="actionReset">
   <property name="icon">
    <iconset resource="synchrotime.qrc">
     <normaloff>:/images/remove.png</normaloff>:/images/remove.png</iconset>
   </property>
   <property name="text">
    <string>Reset</string>
   </property>
   <property name="toolTip">
    <string>Reset the RTC to the default setting</string>
   </property>
  </action>
  <action name="actionSetRegister">
   <property name="icon">
    <iconset resource="synchrotime.qrc">
     <normaloff>:/images/edit.png</normaloff>:/images/edit.png</iconset>
   </property>
   <property name="text">
    <string>Set Register</string>
   </property>
   <property name="toolTip">
    <string>Set a new value to the RTC offset register</string>
   </property>
  </action>
  <action name="actionPort_Setting">
   <property name="icon">
    <iconset resource="synchrotime.qrc">
     <normaloff>:/images/settings.png</normaloff>:/images/settings.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Port Setting</string>
   </property>
   <property name="toolTip">
    <string>Configure serial port</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+P</string>
   </property>
  </action>
  <action name="actionSelect_Font">
   <property name="icon">
    <iconset resource="synchrotime.qrc">
     <normaloff>:/images/font.png</normaloff>:/images/font.png</iconset>
   </property>
   <property name="text">
    <string>Select Font</string>
   </property>
  </action>
  <action name="actionClean_Up">
   <property name="icon">
    <iconset resource="synchrotime.qrc">
     <normaloff>:/images/clear.png</normaloff>:/images/clear.png</iconset>
   </property>
   <property name="text">
    <string>C&amp;lean Up</string>
   </property>
   <property name="toolTip">
    <string>Clean up data</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+L</string>
   </property>
  </action>
  <action name="actionAbout_Qt">
   <property name="text">
    <string>Qt-Framework</string>
   </property>
  </action>
  <action name="actionAbout_App">
   <property name="text">
    <string>About App</string>
   </property>
   <property name="toolTip">
    <string>About program</string>
   </property>
  </action>
  <action name="actionContents">
   <property name="icon">
    <iconset resource="synchrotime.qrc">
     <normaloff>:/images/help.png</normaloff>:/images/help.png</iconset>
   </property>
   <property name="text">
    <string>Help</string>
   </property>
   <property name="shortcut">
    <string>F1</string>
   </property>
  </action>
  <action name="actionEnglish">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>English</string>
   </property>
   <property name="toolTip">
    <string>Select English interface</string>
   </property>
  </action>
  <action name="actionGerman">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>German</string>
   </property>
   <property name="toolTip">
    <string>Select German interface</string>
   </property>
  </action>
  <action name="actionRussian">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Russian</string>
   </property>
   <property name="toolTip">
    <string>Select Russian interface</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>../../qcustomplot/qcustomplot.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Console</class>
   <extends>QWidget</extends>
   <header>console.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="synchrotime.qrc"/>
 </resources>
 <connections/>
</ui>
